files/setup-06-event-router.sh

```diff
@@ -9,6 +9,9 @@ set -euo pipefail
 echo -e "\e[92mDeploying VMware Event Router ..." > /dev/console
 kubectl --kubeconfig /root/.kube/config -n vmware create secret generic event-router-config --from-file=${EVENT_ROUTER_CONFIG}
 
+# Retrieve the version tag for VMware Event Router image
+EVENT_ROUTER_VERSION=$(awk '/Version:/ {print $2}' /etc/veba-release)
```

Comment:
I think that's a good approach since we can always create custom (beta) builds of the appliance/router image which would be reflected in this file as well.